### ISSUE TITLE
Task isn't being completed 

### ISSUE THREAD

**Issue by xNabil** [akasakaid/majtod#4]
![Screenshot_20241009-003401](https://github.com/user-attachments/assets/f672f29d-8386-43ee-9bf5-95393c559ae0)

**most of the free tasks are not being completed** 
by brother uses it on pc he doesn't has this kind of problem,
i am on termux, any solutions for me?

**Comment by akasakaid** [comment]
> ![Screenshot_20241009-003401](https://github.com/user-attachments/assets/f672f29d-8386-43ee-9bf5-95393c559ae0)
> 
> **most of the free tasks are not being completed** 
> by brother uses it on pc he doesn't has this kind of problem,
> i am on termux, any solutions for me?

As I explained in the readme. Not all tasks can be done.

**Comment by xNabil** [comment]
yeah i know, but tasks like `join major telegram ` or `watch YouTube video` is being completed on pc but not on termux 🥲

**Comment by akasakaid** [comment]
Idk dude

**Comment by xNabil** [comment]
> Idk dude

btw brother setting up by query id is very time consuming for multi TG users specially if u are using Android 

NotPixTod was very easy because i can directly login
can't we do the same for BlumTod & MajTod🙂?
